core/templates/components/state-editor/state-editor-properties-services/state-skill.service.ts

```diff
@@ -21,7 +21,7 @@ import { Injectable } from '@angular/core';
 
 import { AlertsService } from 'services/alerts.service';
 import { StatePropertyService } from
-  // eslint-disable-next-line max-len
+  // eslint-disable-next-line
```

Comment:
Do you still need the eslint-disable at all here? Can it be removed entirely?

Reply:
It gives the max-len error without the eslint-disable.

Reply:
Hang on... then, shouldn't you just be disabling that specific rule? I.e.
why is this line changed from before?

Reply:
Ohh right! Done. PTAL!